sentinelsat/sentinel.py

```diff
@@ -835,19 +835,19 @@ def geojson_to_wkt(geojson_obj, feature_number=0, decimals=4):
     else:
         geometry = geojson_obj['features'][feature_number]['geometry']
 
-    def ensure_2d(geometry):
-        if isinstance(geometry[0], (list, tuple)):
-            return list(map(ensure_2d, geometry))
+    def ensure_2d(_geometry):
```

Comment:
why? I have never seen `_private` parameters to functions...

Reply:
Yeah, I was struggling with the name and just used underscore, which misses the point.

Reply:
I see. What does `geometry` conflict with?

Reply:
With geometry from the outer function, it's not fatal, but in my opinion it's at least bad taste, at worst may lead to problems in future.